### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by skeydan**
thanks for reviewing this! I've added a new tag "introductions", do you think that's a good idea?

### CODE REVIEW

File: _posts/2018-10-11-activations-intro/activations_intro.Rmd [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+```{r}
+model <- keras_model_sequential() %>%
+  layer_dense(units = 32) %>%
+  layer_dense(units = 1)
+
+model %>% compile(optimizer = "adam", loss = "mean_squared_error")
+
+model %>% fit(
+  x = iris$Sepal.Length %>% as.matrix(),
```
Comment (jjallaire) [inline _posts/2018-10-11-activations-intro/activations_intro.Rmd]: AFAIK we shouldn't need the `as.matrix()` here.
